docs/source/download.rst

```diff
@@ -1,111 +1,134 @@
 
-.. _Docker: https://www.docker.com/  
+venv route
+``````````
 
-.. code-block:: shell
+This is the recommended method when setting up Devito as part of a larger project
+that uses Devito among other python packages.
+Devito is available as a `pip package`_ in PyPI.
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
 
-   # 1. run the tests
-   docker-compose run devito /tests
+Create a `Python virtual environment`_
 
-   # 2. start a jupyter notebook server on port 8888
-   docker-compose up devito
+.. _Python virtual environment: https://docs.python.org/3/library/venv.html
 
-   # 3. start a bash shell with devito
-   docker-compose run devito /bin/bash
+.. code-block:: shell
 
-The three sample commands above have only been included as illustrations of typical
-uses of Devito inside a container. They are not required to be run in that order.
+  python3 -m venv <your_venv_name>
 
-1. Command 1 above runs the unit tests included with Devito to check whether the
-installation succeeded. This is not necessary but a handy first thing to run. As
-long as the first few tests are passing, it is possible to press `Ctrl+C` to stop
-the testing and continue.
+Source the newly created `venv`. This needs to be repeated each time a new terminal is open.
 
-2. Command 2 starts a Jupyter_ notebook server inside the
-docker container and forwards the port to `http://localhost:8888`.
-After running this command, you can copy-paste the complete URL from the terminal window where
-the command was run - to a browser to open a jupyter session to try out the included
-tutorials. Alternatively, you may simply point your browser to `http://localhost:8888`
-and, if prompted for a password, copy-paste the authentication token from the command
-window. Once successfully in the Jupyter notebook session, proceed to run the tutorials
-provided in the `examples` folder or create your own notebooks. 
+.. code-block:: shell
 
-3. Command 3 above starts a bash (command-line) shell with Devito loaded into the
-environment. Essentially, it means that any python code run after this command will
-see devito when doing imports like `from devito import Function`. Any code using
-Devito can be executed using this method.
+  source <your_venv_name>/bin/activate
 
-Issues? Ask us for help on Slack_. Also look at our list of known
-installation issues_.
 
-.. _Jupyter: https://jupyter.org/
+To install the latest Devito release along with any additional dependencies, follow:
 
-Pip installation
-----------------
-This is the recommended method when setting up Devito as part of a larger project
-that uses Devito among other python packages. To install Devito using `pip`, simply
-do:
+.. code-block:: shell
 
+   pip install devito
+   # ...or to install additional dependencies:
+   pip install devito[extras,mpi,nvidia]
+
+To install the latest Devito development version (current GitHub master) (without the tutorials), follow:
```

Comment:
why "without the tutorials" ? the second line does bring them in no?

Reply:
I do not think so:
https://github.com/devitocodes/devito/blob/master/setup.py#L42

Also: https://github.com/devitocodes/devito/issues/1954